utmap-client/src/components/pages/SingleEventPage.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import {
+	Avatar,
+	Collapse,
+	IconButton,
+	Grid,
+	GridList,
+	makeStyles,
+	Paper,
+	Typography,
+} from '@material-ui/core';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import CloseIcon from '@material-ui/icons/Close';
+import EditIcon from '@material-ui/icons/Edit';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(theme => ({ //CSS styles on components
+	box: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		backgroundColor: 'gray',
+	},
+	paper: {
+		width: '35rem',
+		marginTop: theme.spacing(6),
+		paddingTop: theme.spacing(4),
+		paddingBottom: theme.spacing(4),
+		paddingLeft: theme.spacing(4),
+		paddingRight: theme.spacing(4),
+		display: 'flex',
+	},
+	largeIcon: {
+		width: theme.spacing(6),
+		height: theme.spacing(6),
+	},
+	largeAvatar: {
+		background: '#002884',
+		width: theme.spacing(10),
+		height: theme.spacing(10),
+	},
+	row: {
+		width: '100%',
+		display: 'flex',
+		alignItems: 'center',
+	},
+	title: {
+		textAlign: 'center',
+	},
+
+}));
+
+function SingleEventPage({startDate, endDate, title, description, location, sublocation, organizer}) {
+	const classes = useStyles();
+
+	//expand for description
+	const [expanded, setExpanded] = React.useState(false);
```

Comment:
Instead of `React.useState()` change the import on line 1 to `import React, { useState } from 'react';`. Then you can change this line to `useState(false)`.